.github/README.md

```diff
@@ -49,12 +56,18 @@ That's it! Let's get Collie installed ⤵️
 
 ## 🕹 Install and Usage
 
-**Unix-based systems**
+We have an easy-to-use install script prepared that will install Collie for you in under a minute. To use it, run the command below depending on your operating system
```

Comment:
t: I would maybe add a message like "please check the content of the file before executing that way, to make sure you are executing save code" to signal we take security seriously. As this could also lead to malicious code execution.